ui/src/app/shared/history.ts

```diff
@@ -5,20 +5,24 @@ import {Utils} from './utils';
  * Return a URL suitable to use with `history.push(..)`. Optionally saving the "namespace" parameter as the current namespace.
  * Only "truthy" values are put into the query parameters. I.e. "falsey" values include null, undefined, false, "", 0.
  */
-export const historyUrl = (path: string, params: {[key: string]: any}) => {
+export function historyUrl(path: string, params: {[key: string]: any}) {
```

Comment:
Oh good point! I forgot it had unit tests.

That being said I do think this function should be refactored a bit, so it may not be worth adding more tests at this time for something that will change. I intended `extraSearchParams` as more a hacky interim workaround.

Should definitely revamp the tests when this gets refactored!